README.md

```diff
@@ -20,16 +20,13 @@ This project uses Swift Package Manager to pull in the frameworks from [axe-devt
 
 ### For Manual Testing
 
-1. Add your Deque API Key to `Login.swift`.
-_This app is ready for manual testing out of the box; running it in Xcode will show the floating action button._
+1. To manually test this app, use the [Mobile Analyzer](https://github.com/dequelabs/axe-devtools-mobile-analyzer).
```

Comment:
Do we consider using the analyzer manual testing? Or just another automated testing scenario with a different product?

Reply:
That's covered under step 6 in the Analyzer setup guide.

Reply:
I wasn't saying that it wasn't considered manual testing, just confirming because in my head I think of something different when I hear 'manual testing' (I think of our old setup where you had to tap the FAB), so I just wanted to confirm, that's all. But given what Kim said about this being 'manual from customer's perspective'  it makes sense to me.

Reply:
Sounds good